spec/models/office_spec.rb

```diff
@@ -114,21 +145,71 @@
     end
 
     describe 'telephone number' do
-      context 'when not present' do
-        before { office.telephone_number = nil }
+      # See http://www.area-codes.org.uk/formatting.php#programmers for background info
 
-        it { is_expected.to_not be_valid }
-      end
+      context 'invalid inputs' do
+        context 'when not present' do
+          before { office.telephone_number = nil }
 
-      context 'when badly formatted' do
-        before { office.telephone_number = 'not-valid' }
+          it { is_expected.to_not be_valid }
+        end
 
-        it { is_expected.to_not be_valid }
+        context 'when the input characters other than spaces or numbers' do
+          before { office.telephone_number = 'not-numbers-or-spaces' }
+
+          it { is_expected.to_not be_valid }
```

Comment:
That is currently true, yes, but future changes might change that. I would feel more comfortable if we were explicitly checking for the right error.